src/app/components/Toolbar/Features/Account/ManageableAccountDetails.tsx

```diff
@@ -0,0 +1,162 @@
+          disabled={!wallet.privateKey}
+          onClick={() => setLayerVisibility(true)}
+        />
+      </Box>
+      {layerVisibility && (
+        <LayerContainer hideLayer={hideLayer}>
+          <Tabs alignControls="start">
+            <Tab title={t('toolbar.settings.exportPrivateKey.title', 'Export Private Key')}>
+              <Box
+                flex="grow"
+                justify="between"
+                height={{ min: isMobile ? 'auto' : layerOverlayMinHeight }}
+                pad={{ vertical: 'medium' }}
+              >
+                <Box gap="medium">
+                  <Text>
+                    {t(
+                      'toolbar.settings.exportPrivateKey.hint1',
+                      'The private key consists of a string of characters, and owning the private key is equivalent to owning the asset ownership.',
+                    )}
+                  </Text>
+                  <Text>
+                    {t(
+                      'toolbar.settings.exportPrivateKey.hint2',
+                      'Once the private key is lost, it cannot be retrieved. Please make sure to Backup the private key and keep it in a safe place.',
+                    )}
+                  </Text>
+                </Box>
+                {!acknowledge && !passwordCheckPass && (
+                  <Button
+                    alignSelf="center"
+                    primary
+                    label={t(
+                      'toolbar.settings.exportPrivateKey.confirm',
+                      'I understand, reveal my private key',
+                    )}
+                    onClick={() => setAcknowledge(true)}
+                  />
+                )}
+
+                {acknowledge && !passwordCheckPass && (
+                  <Form onSubmit={onSubmit} {...preventSavingInputsToUserData}>
+                    <PasswordField
+                      placeholder={t('persist.loginToProfile.enterPasswordHere', 'Enter your password here')}
+                      name="password"
+                      inputElementId="password"
+                      autoFocus
+                      value={password}
+                      onChange={event => setPassword(event.target.value)}
+                      error={
+                        enteredWrongPassword
+                          ? t('persist.loginToProfile.wrongPassword', 'Wrong password')
+                          : false
+                      }
+                      showTip={t('persist.loginToProfile.showPassword', 'Show password')}
+                      hideTip={t('persist.loginToProfile.hidePassword', 'Hide password')}
+                      width="auto"
+                    />
+                    <Box direction="row" justify="end" margin={{ top: 'medium' }}>
+                      <Button
+                        primary
+                        type="submit"
+                        label={t('toolbar.settings.exportPrivateKey.validate', 'Validate password')}
```

Comment:
I would replace this with "I understand, reveal my private key", and skip a step

Reply:
You mean remove the initial "I understand, reveal my private key" step and just show pass input right away? Ok.